kubernetes/loculus/templates/ena-submission-config.yaml

```diff
@@ -4,6 +4,7 @@
 {{- $submitToEnaProduction := .Values.submitToEnaProduction | default false }}
 {{- $enaDbName := .Values.enaDbName | default false }}
 {{- $enaUniqueSuffix := .Values.enaUniqueSuffix | default false }}
+{{- $enaLogLevel := .Values.enaLogLevel | default "INFO" }}
```

Comment:
I just realized these aren't required - @corneliusroemer you showed me how to set this in the values.yaml already so I will remove before merging.